ci/azure-pipelines-docs-integration.yml

```diff
@@ -58,14 +56,15 @@ stages:
                 docs/**
                 tests/integration/docusaurus/**
                 tests/integration/fixtures/**
+                tests/integration/test_definitions/**
+                tests/integration/test_script_runner.py
```

Comment:
Nice catch!

Reply:
Yea for sure... it's nice to manually trigger when needed

Reply:
But we are gonna talk about a bunch of potential pipeline changes at the arch meeting tomorrow. I think we should make all/most pipelines and stages manually runnable.